__tests__/reducers/index.test.js

```diff
@@ -82,6 +83,7 @@ describe(`Takes a resource template ID and populates the global state`, () => {
   })
 
   it('handles SET_RESOURCE_TEMPLATE', () => {
+    const testId = jest.spyOn(shortid, 'generate').mockReturnValue(0)
```

Comment:
this is set but never used.  Did you forget to add an expect statement, or did you forget to remove it?